src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -196,6 +196,24 @@ export const selectPortfolioCryptoHumanBalanceByAssetId = createSelector(
     fromBaseUnit(bnOrZero(balances[assetId]), assets[assetId]?.precision ?? 0)
 )
 
+export const selectPortfolioCryptoHumanBalanceBySymbol = createSelector(
```

Comment:
That works since gem doesn't use [CAIPs](https://github.com/shapeshift/caips) to identify assets, but please add it to the risks sections of your PR.
Identifying assets like this might cause collisions with assets that have the same symbol, but are actually different ones on the same chain, or even on a totally different chain.